### PR TITLE
Added requirements.txt

### PR DESCRIPTION
**Pull request by gbenhaim**
This file contains all the deps
that can be downloaded from PyPi.

Other deps should be downloaded with
the platform's native package manager.

Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

**Comment by gbenhaim** [comment]
Part of #309

**Comment by nvgoldin** [comment]
:+1: 
I guess libvirt-python is not working properly on ubuntu from PIP?

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+lxml
+lockfile
+paramiko
+pbr
```
Comment (pilou-) [inline requirements.txt]: [`dulwich`](https://github.com/lago-project/lago/blob/adding_requirements.txt/scripts/version_manager.py#L10) (used in  `scripts/version_manager.py` which is called by [`setup.py`](https://github.com/lago-project/lago/blob/adding_requirements.txt/setup.py#L37)), [`file-magic`](https://github.com/lago-project/lago/blob/adding_requirements.txt/lago/templates.py#L26) and `libvirt-python` are missing.

**Comment by nvgoldin** [comment]
dulwich - is used only for building/testing as far as I know.
file-magic - is taken from the RPM(python-magic). What that is available on PyPi is a different package.

**Comment by gbenhaim** [comment]
@nvgoldin 
You are getting it for free when installing libvirt-bin :)

**Comment by pilou-** [comment]
`pbr` is used only for building, but this package is listed: either both `pbr` and `dulwich` should be listed or both should not.

[file-magic ](https://pypi.python.org/pypi/file-magic/0.3.0) pip package contains [magic.py](https://github.com/file/file/blob/master/python/magic.py) released in file 5.26.

**Comment by gbenhaim** [comment]
Removed pbr

**Comment by gbenhaim** [comment]
ci merge please

**Pull request merged**